src/Tizen.NUI/src/public/InputMethodContext.cs

```diff
@@ -654,6 +654,19 @@ public void SetMIMEType(string mimeType)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Sets the x,y coordinates of the input panel.
+        /// </summary>
+        /// <param name="x">The top-left x coordinate of the input panel.</param>
+        /// <param name="y">The top-left y coordinate of the input panel.</param>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
could you change tizen_6.0 as tizen_next ?

Reply:
Okay, changed to 'tizen_next' in the description